src/components/SyntaxHighlighter/YqlHighlighter.tsx

```diff
@@ -0,0 +1,25 @@
+import {PrismLight as SyntaxHighlighter} from 'react-syntax-highlighter';
+
+import {b} from './shared';
+import {useSyntaxHighlighterStyle} from './themes';
+import {yql} from './yql';
+
+SyntaxHighlighter.registerLanguage('yql', yql);
+
+interface YqlHighlighterProps {
+    children: string;
+    className?: string;
+}
+
+/** SyntaxHighlighter with just YQL for sync load */
+export function YqlHighlighter({children, className}: YqlHighlighterProps) {
```

Comment:
Previous `YqlHighlighter` with little changes to render queries in tables and overview. The reason - we load a lot of other languages in `YDBSyntaxHighlighter`, so it's loaded with lazy load. However, in tables lazy load works not very good, since there are may loaders and row height changes. So to make it lazy in tables we need first create some loader for queries - it will display query with proper size but without highlight